### DISCUSSION TITLE
How do you disable the sound?

### DISCUSSION THREAD

**Discussion by timofey-drozhzhin** [rsayfutdinov/Pikka#49] [Q&A]
Is it me or is there no way to disable the sound effect on the color picker?

I usually wear headphones, so the sound is pretty sharp. With the color picker, I usually trigger 10-20 dings a minute, making my brain go nuts! 😬

**Comment by rsayfutdinov** [comment]
That's right!:) I will add an option to settings, to disable the sound.

**Reply by timofey-drozhzhin** [reply]
Oh great, thank you!

**Reply by rsayfutdinov** [reply]
Update is already in AppStore
